src/containers/FormikForm/TitleField.tsx

```diff
@@ -20,17 +19,10 @@ interface Props {
   maxLength?: number;
   name?: string;
   handleSubmit: () => Promise<void>;
-  onBlur: (event: Event, editor: Editor) => void;
   type?: string;
 }
 
-const TitleField = ({
-  t,
-  maxLength = 256,
-  name = 'slatetitle',
-  handleSubmit,
-  onBlur,
-}: Props & tType) => {
+const TitleField = ({ t, maxLength = 256, name = 'slatetitle', handleSubmit }: Props & tType) => {
```

Comment:
No som alle titler i forms bruker slate kan vi endre denne tilbake til `title`. Navnet her var kun for å skille mellom dei to variantene.